audio/deepvoice3/dependencies/text/symbols.py

```diff
@@ -0,0 +1,17 @@
+'''
+Defines the set of symbols used in text input to the model.
+
+The default is a set of ASCII characters that works well for English or text that has been run
+through Unidecode. For other data, you can modify _characters. See TRAINING_DATA.md for details.
```

Comment:
Remove references to TRAINING_DATA.md.
Also, if these files are borrowed from the source repository, please include a copyright disclaimer in the beginning.
i.e. "This file was originally created by the developers of the DeepVoice3 repository and is used to preprocess input data."

Reply:
Made the change and also removed the pycache files.